### PR TITLE
[MSHARED-1002] Require Java 8

### PR DESCRIPTION
**Pull request by slachiewicz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.maven.shared</groupId>
     <artifactId>maven-shared-components</artifactId>
-    <version>34</version>
+    <version>36</version>
```
Comment (slawekjaranowski) [inline pom.xml:25]: maybe separate PR and issue

Comment (slachiewicz) [reply pom.xml:25]: I don't think it would have value, first i updated java and then updated pom. In pom.xml we have almost nothing

**Review by slawekjaranowski — APPROVED** [review]
Please look in Jenkins ... what is wrong with connected builds

**Comment by jira-importer** [comment]
Resolve #145

**Pull request merged**